traveler_app/src/main/java/com/guestlogix/traveler/network/GuestRoute.java

```diff
@@ -16,9 +16,13 @@
 import static com.guestlogix.travelercorekit.utilities.TravelerPrefs.Key.TRAVELER_AUTH_ENDPOINT;
 import static com.guestlogix.travelercorekit.utilities.UrlHelper.urlEncodeUTF8;
 
-class GuestRoute {
+public class GuestRoute {
```

Comment:
this opens GuestRoute class visibility, but all of its useful parts are private or package-private anyway